addons/mail/static/src/components/thread_view_topbar/thread_view_topbar.scss

```diff
@@ -11,7 +11,7 @@
     height: $font-size-sm;
 }
 
-.o_ThreadViewTopbar_threadName {
+.o_ThreadViewTopbar_threadName, .o_ThreadViewTopbar_threadDescription {
```

Comment:
We try to avoid `,` rules, instead create a new class common to both whenever that is needed.